### ISSUE TITLE
[dispatch] handle: failed to parse S_LOGIN<1>

### ISSUE THREAD

**Issue by DaXmAn** [meishuu/tera-proxy#60]
I updated  tera-data, tera-data-parser, tera-proxy-game and tera-proxy-sls directly from respecive repositories but I still get this error: [dispatch] handle: failed to parse S_LOGIN<1> for timestamp module I'm trying to use (https://github.com/wuaw/timestamps)

**Comment by Snugglez** [comment]
S_LOGIN v1 has been deprecated for a while, either find the old def or update the module your self. Also recommend not using meishus version of proxy all together since it has been unmaintained for a while now, use caali or pinkipi's fork.

**Comment by DaXmAn** [comment]
Thanks for response, you mean this repository? https://github.com/hackerman-caali

**Comment by Snugglez** [comment]
Yes, for any help and info on his version feel free to join https://discord.gg/maqBmJV and check out the respective channels for it
